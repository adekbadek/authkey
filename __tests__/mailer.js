const {getMailOptions} = require('../src/mailer')

it('getMailOptions', () => {
  const mailOptions = getMailOptions({
    to: 'someone@maillcom',
    config: {
      from: 'someone@mail.com',
      productName: process.env.PRODUCT_NAME,
    },
    authkey: '72389y4cr8gf23',
  })
  expect(mailOptions).toMatchSnapshot()
})
