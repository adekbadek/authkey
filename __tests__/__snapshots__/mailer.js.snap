// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getMailOptions 1`] = `
Object {
  "from": "someone@mail.com",
  "html": "Your auth key for MyProduct is <strong>72389y4cr8gf23</strong>.",
  "subject": "Auth key for MyProduct",
  "text": "Your auth key for MyProduct is 72389y4cr8gf23.",
  "to": "someone@maillcom",
}
`;
